cmd/validate.go

```diff
@@ -40,15 +43,21 @@ var (
 			if err := config.SetProjectRoot(args); err != nil {
 				exit(err, cmd.UsageString())
 			}
+			installMissingLanguagePlugin(installPlugins)
 			validation.Validate(args)
 		},
 		DisableAutoGenTag: true,
 	}
 	hideSuggestion bool
 )
 
+func installMissingLanguagePlugin(flag bool) {
```

Comment:
Can we use the existing `installMissingPlugins` method with addition flag param `languageOnly`? I guess that will remove all the new methods which you have added.

Reply:
Yeah agreed. But I need the install.LanguagePlugin thing or else I need to pass it around everywhere that might create a mess. Like why AllPlugins should take a flag called languageOnly

Reply:
> I need to pass it around everywhere

Isn't it better than repeating 3 or more functions?